R/get_connection.R

```diff
@@ -172,15 +173,29 @@ get_nuvolos_db_path <- function() {
 #' 
 #' Using outside of Nuvolos only. This helps the user to store credentials safely at local device.
 #' @export
-input_nuvolos_credential <- function(username, password){
+input_nuvolos_credential <- function(){
   # store username & password
+  username <- rstudioapi::askForSecret("Please input your Nuvolos username:")
   keyring::key_set_with_value("nuvolos", "username", username)
+  
+  password <- rstudioapi::askForSecret("Please input your Nuvolos password:")
```

Comment:
don't you need to specify that this is the message /  title?

https://www.rdocumentation.org/packages/rstudioapi/versions/0.11/topics/askForSecret

Reply:
change it now. thx for catching this.